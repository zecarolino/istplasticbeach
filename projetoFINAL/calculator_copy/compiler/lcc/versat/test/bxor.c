int x = 5;
int main() { return 6 ^ x; }
