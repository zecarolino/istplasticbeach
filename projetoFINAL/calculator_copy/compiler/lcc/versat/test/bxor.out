VCD info: dumpfile xtop.vcd opened for output.
3
Catched Trap at time  393 clock cycles:
